test/test_tensorrec.py

```diff
@@ -8,6 +8,7 @@
 from tensorrec import TensorRec
 from tensorrec.util import generate_dummy_data_with_indicator, generate_dummy_data
 from tensorrec.session_management import set_session
+from test.datasets import get_movielens_100k
```

Comment:
I'm not sure if using the movielens data in `test_tensorrec` is un-koshier. Let me know if there's a better way to lay these out.

Reply:
Do we get non-zeros with movielens but zeros with the random data? That would really surprise me.

I'm concerned about using movielens only because it makes the test quite a bit slower.

Reply:
https://media.giphy.com/media/l3q2K5jinAlChoCLS/giphy.gif

Reply:
Ok I played around with it and was actually able to get some random data with non-zero biases. Right now it's on really small sample sizes but let me see if I can change the `standard_model` so it doesn't break other tests but also has a model now with more Real Biases™.

Reply:
Ok, I've figured it out. It's not the random data itself. Something else is going on. It seems that, if `TensorRec#fit` is called more than once in the same session, the biases are cleared out. I don't know why this is happening. It doesn't matter if they are different instances of `TensorRec`.

At first I thought this was an issue of `fit` being called multiple times on the same instance, so I changed `TensorRecTestCase#setUpClass` to `setUp` so that it would run before each test method, but with the same result.

Next, in `TensorRecTestCase#setUp`, I separated out the data sources of `standard_model` and `unbiased_model` completely, and printed the returned biases after the 1st `fit` call and after the 2nd `fit` call:

```python
    def setUp(cls):
        cls.interactions, cls.user_features, cls.item_features = generate_dummy_data(
            num_users=15, num_items=30, interaction_density=.5, num_user_features=200, num_item_features=200,
            n_features_per_user=20, n_features_per_item=20, pos_int_ratio=.5
        )

        cls.unbiased_interactions, cls.unbiased_user_features, cls.unbiased_item_features = generate_dummy_data(
            num_users=15, num_items=30, interaction_density=.5, num_user_features=200, num_item_features=200,
            n_features_per_user=20, n_features_per_item=20, pos_int_ratio=.5
        )

        cls.standard_model = TensorRec(n_components=10)
        cls.standard_model.fit(cls.interactions, cls.user_features, cls.item_features, epochs=10)

        print(cls.standard_model.predict_item_bias(cls.item_features))

        cls.unbiased_model = TensorRec(n_components=10, biased=False)
        cls.unbiased_model.fit(cls.unbiased_interactions, cls.unbiased_user_features, cls.unbiased_item_features,
                               epochs=10)

        print(cls.standard_model.predict_item_bias(cls.item_features))
```

This prints out

```
[ 1.1775156  -0.5886802  -0.70887476  0.19068862  0.24826032 -0.16348793
  0.3098605   0.32488823  0.62933105 -0.34945598  0.254797   -0.5822147
 -0.32148573  0.15126142 -0.70438     0.31067222  0.40411028  0.77066743
 -0.71558595  0.3316024  -0.10362635  0.72578794  0.8436037  -0.3284939
  0.70730585  1.3022892  -0.8598295   0.9130077   0.15513004  0.66234463]
[0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0.
 0. 0. 0. 0. 0. 0.]
```

i.e. the data we want, then the zeroed-out data. I'm not sure why this is happening yet. Logically, it seems most likely to be tied to session sharing, since that's the only thing I see being shared between instances.

This won't be a problem for the tests I have to write, as I can just create a new test class with just a `standard_model` and be on my merry way, but it's probably an issue that someone will run into in the future.